gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/couchdb_request_time.json

```diff
@@ -0,0 +1,40 @@
+{
+  "display_name": "CouchDB request time stays within 100ms",
+  "combiner": "OR",
+  "conditions": [
+    {
+      "condition_threshold": {
+        "filter": "metric.type=\"custom.googleapis.com/couchdb/httpd_request_time\" resource.type=\"gke_container\"",
+        "comparison": "COMPARISON_GT",
+        "threshold_value": 100,
```

Comment:
I checked production data and it looks like in worst cases CouchDB manages to respond in less than 50ms. I doubled this value to have some buffer and used 100ms as a threshold.

Reply:
This is useful information for operators looking at this alert. Can we make it visible somewhere? Maybe in the `documentation` stanza (this file doesn't have one, but the other file does)?